### DISCUSSION TITLE
How To: preview colors in the terminal

### DISCUSSION THREAD

**Discussion by chipbuster** [starship/starship#1894] [Ideas]
Starship's [color configuration](https://starship.rs/config/#style-strings) takes many forms, but it can sometimes be tricky to know *exactly* what color you want before you try it out. What tricks do y'all have for previewing/selecting colors in the terminal?

**Comment by Mgldvd** [comment]
## View ANSI colors

https://starship.rs/config/

```toml
[package]
format = "[⬢ $version](94) "
```

https://starship.rs/config/#style-strings

With this snippet(alias) you will be able to see all the colors in the terminal.

Place on your `~/.bashrc` or `~/.zshrc`

```bash
colors-256 () {
    for (( i = 0; i < 256; i++ )); do
        echo -e "\e[38;5;${i}m ██ Color: ${i}  ······· use: echo -e \"\\\e[38;5;${i}m TEXT\""
    done
}
```

Then open your terminal and run `colors-256`

![Selection_008](https://user-images.githubusercontent.com/2009698/99323910-1f6aa480-2841-11eb-9f54-926a56ede7d0.png)
